libs/plustek-sdk/src/config.ts

```diff
@@ -0,0 +1,84 @@
+export interface Config {
+  savepath?: string
+  'filename-format'?: string
+  'filename-beginindex'?: number
+  'paper-size'?:
+    | 'A3'
+    | 'A4'
+    | 'A5'
+    | 'B5'
+    | 'B6'
+    | 'Letter'
```

Comment:
+ 'Legal' ?

Reply:
These are all that were listed in the official SDK. However, the paper size selected seems to have no effect as it'll scan Letter or Legal with "Letter" size selected.